PyDSS/pydss_project.py

```diff
@@ -370,7 +377,7 @@ def _zip_project_files(self, delete=True):
             os.chdir(orig)
 
     @staticmethod
-    def load_simulation_config(project_path):
+    def load_simulation_config(project_path, scenario):
```

Comment:
I think there might be a problem here.  At the call site `scenario` can be None, so the `os.path.join()` with a None wouldn't make sense.

Also, same comment as earlier, `simulation_file` would be more clear.  Perhaps you could use
```
def load_simulation_config(project_path, simulation_file=SIMULATION_SETTINGS_FILENAME):
```

Reply:
scenario changed to simulation_file. This should not be none given I set the default value from the common.py file. -s overrides the default. thoughts?

Reply:
I see what you're saying.  The cli/run function sets `scenario=None` but the default click value is SIMULATION_SETTINGS_FILENAME, so it should never be None.  I got tricked by the run function signature. I suggest changing it to `simulation_file=SIMULATION_SETTINGS_FILENAME`

Reply:
fixed